.github/workflows/standalone.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       run: pip install .[test]
 
     - name: Install pyinstaller
-      run: pip install pyinstaller==6.6
+      run: pip install pyinstaller<6
```

Comment:
I think you have to pin this for the OSX job.

Reply:
Good call. I also wanted to see if it breaks the non-Mac job, but I guess there's nothing stopping us from using separate versions now that the two jobs are separate.